modules/persistent-storage-csi-snapshots-provision.adoc

```diff
@@ -5,14 +5,14 @@
 [id="persistent-storage-csi-snapshots-provision_{context}"]
 = Provisioning a volume snapshot
 
-There are two ways to provision snapshots: xref:#snapshots-manual-provisioning[manually] and xref:#snapshots-dynamic-provisioning[dynamically].
-
-[id="snapshots-manual-provisioning"]
-== Manual provisioning
-
-As a cluster administrator, you can manually pre-provision a number of VolumeSnapshotContent objects. These carry the real volume snapshot details available to cluster users.
+There are two ways to provision snapshots: xref:#snapshots-dynamic-provisioning[dynamically] and xref:#snapshots-manual-provisioning[manually].
```

Comment:
The xref ban extends to internal xrefs.

Reply:
Ah, okay. Was thinking internal was possible. Removing xrefs.